SmartDeviceLink/SDLLockScreenManager.m

```diff
@@ -27,7 +28,15 @@ @interface SDLLockScreenManager ()
 @property (assign, nonatomic) BOOL canPresent;
 @property (strong, nonatomic, readwrite) SDLLockScreenConfiguration *config;
 @property (strong, nonatomic) id<SDLViewControllerPresentable> presenter;
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
 @property (strong, nonatomic, nullable) SDLOnLockScreenStatus *lastLockNotification;
+#pragma clang diagnostic pop
+
+@property (strong, nonatomic, nullable) SDLOnDriverDistraction *lastDriverDistractionNotification;
+@property (assign, nonatomic, readwrite, getter=isLockScreenDismissable) BOOL lockScreenDismissable;
+@property (assign, nonatomic) BOOL lockScreenDismissed;
```

Comment:
```suggestion
@property (assign, nonatomic) BOOL lockScreenDismissedByUser;
```

Reply:
This does not appear to be set to `NO` anywhere outside of initialization. As a result, I don't think that this line in the proposal is implemented.

> If a subsequent notification is received that changes the lockScreenDismissalEnabled parameter, the lockscreen will re-appear.

So if the lock screen is dismissed, and the driver is distracted, and the `lockScreenDismissalEnabled` is `NO`, then the lock screen needs to be presented again but with the dismissal gesture disabled. It appears to only be disabling the dismissal gesture on L201-202.